src/idpi/operators/omega_slope.py

```diff
@@ -30,6 +27,8 @@ def omega_slope(
     ps: xr.DataArray, etadot: xr.DataArray, ak: xr.DataArray, bk: xr.DataArray
 ):
     """Compute omega slope."""
+    surface_pressure_ref = 101325.0
+
```

Comment:
How come this is not in the physical_constants file?

Reply:
Seems more like reference value rather than a physical constant but I can move it there if you prefer.

Reply:
I think its better in the constant file, it coud be useful somewhere else too.